css-canvas-medical/styles.css

```diff
@@ -0,0 +1,170 @@
+* {
+  font-family: Avenir, sans-serif;
+  font-style: normal;
+}
+
+.navi {
+  margin-right: 50px;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-left: 25px;
+  margin-right: 25px;
+  margin-top: 15px;
+}
+
+.nav-container {
+  display: flex;
+  justify-content: space-between;
+
+  /* border: 1px solid red; */
+  width: 100%;
+  border-bottom: 1px solid rgba(119, 119, 119, 0.342);
+  padding-bottom: 15px;
+}
+
+.img-logo {
+  width: 125px;
+  margin-left: 50px;
+}
+
+a {
+  margin: 20px;
+  text-decoration: none;
+  color: rgb(119, 119, 119);
+}
+
+button {
+  background-color: rgb(41, 68, 100);
+  color: white;
+  padding: 6px;
+  border-radius: 5px;
+}
+
+.hero-jpg {
+  width: 1600px;
+  height: 800px;
+  background-image:
+    linear-gradient(to bottom, rgba(245, 246, 252, 0), rgba(49, 44, 48, 0.774)),
+    url(./images/hero.jpg);
+  background-repeat: no-repeat;
+  margin-top: 55px;
+  color: white;
+}
+
+.pcare {
+  width: 45%;
+  line-height: 35px;
+  font-size: 25px;
+}
+
+.emr-title {
+  margin-bottom: -20px;
```

Comment:
You should definitely avoid negative values in margin. What is the purpose of this? It can likely be dealt with another way.

Reply:
Oh okay I just used line height instead.